samples/wrappedBinding/hnApi.js

```diff
@@ -0,0 +1,15 @@
+// This function will be called to wrap inner bindings passed as env argument.
+// The return value is used as a resulting binding value.
+export function wrapBindings(env) {
+  return {
+    async query(q) {
+      let url = "http://hn.algolia.com/api/v1/search_by_date?";
```

Comment:
Good idea to add a comment here reminding folks that this points to an external server whose content is out of our control.